test/python/test_tools_import_special_phrases.py

```diff
@@ -0,0 +1,120 @@
+"""
+    Tests for import special phrases functions
+"""
+from pathlib import Path
+import pytest
+from nominatim.tools.special_phrases import SpecialPhrasesImporter
+
+TEST_BASE_DIR = Path(__file__) / '..' / '..'
+
+def test_process_amenity_with_operator(special_phrases_importer, getorcreate_amenityoperator_funcs):
+    special_phrases_importer._process_amenity('', '', '', '', 'near')
+    special_phrases_importer._process_amenity('', '', '', '', 'in')
```

Comment:
You'd still want to check that the expected SQL functions got executed at all. You just don't need to care what the SQL functions themselves do. You can do that by still having a simple 'INSERT .. INTO' into some custom database table and then check that there is a new line in that database table.